flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/configurable_variables_v2_get/config/config6.json

```diff
@@ -374,7 +374,7 @@
     "ballerina/log": {
       "": [
         {
-          "id": "94278",
+          "id": "113165",
```

Comment:
@NipunaRanasinghe can you please check and verify the changes in this test file?